fw/Core/Hitcon/App/TetrisApp.cc

```diff
@@ -0,0 +1,85 @@
+#include "TetrisApp.h"
+
+#include <App/MenuApp.h>
+#include <Logic/BadgeController.h>
+#include <Logic/Display/display.h>
+#include <Logic/RandomPool.h>
+#include <Service/Sched/SysTimer.h>
+#include <Service/Sched/Task.h>
+
+using hitcon::service::sched::SysTimer;
+using hitcon::service::sched::task_callback_t;
+
+namespace hitcon {
+
+TetrisApp tetris_app;
+
+TetrisApp::TetrisApp()
+    : periodic_task(hitcon::tetris::UPDATE_PRIORITY,
+                    (task_callback_t)&TetrisApp::periodic_task_callback, this,
+                    hitcon::tetris::UPDATE_INTERVAL),
+      game([]() { return g_fast_random_pool.GetRandom(); }) {
+  hitcon::service::sched::scheduler.Queue(&periodic_task, nullptr);
```

Comment:
Would recommend queueing with an Init() function because scheduler may not be available at construction time.